pkg/kobject/kobject.go

```diff
@@ -107,3 +108,15 @@ type Ports struct {
 	HostIP        string
 	Protocol      api.Protocol
 }
+
+// Volumes holds the volume struct of container
+type Volumes struct {
+	SvcName    string
+	MountPath  string
+	VFrom      string
+	VolumeName string
+	Host       string
+	Container  string
```

Comment:
`mountpath` can be `host:container`, hence `container` is that directory, for example, `/abc`